packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -390,40 +401,13 @@ export async function postTransformationJob() {
 }
 
 export async function transformationJobErrorHandler(error: any) {
-    if (transformByQState.isCancelled()) {
```

Comment:
Removed because the "stopJob" API was being called twice in some circumstances -- when user clicks stop that is already handled.